group changes webhooks and pushes (#929)
livechat/livechat-public-docs: content/management/configuration-api/v3.4/index.mdx

| **Thread tags**   | [`thread_tagged`](#thread_tagged) [`thread_untagged`](#thread_untagged)                                                                                                                                                                                                                                                                         |
| **Status**        | [`routing_status_set`](#routing_status_set) [`agent_deleted`](#agent_deleted)                                                                                                                                                                                                                                                                   |
| **Customers**     | [`incoming_customer`](#incoming_customer) [`customer_session_fields_updated`](#customer_session_fields_updated)                                                                                                                                                                                                                                 |

> | **Configuration** | [`group_create`](#group_created) [`group_updated`](#group_updated) [`group_deleted`](#group_deleted)                                                                                                                                                                                                                                            |

Reviewer: Typo

Reviewer 2: Yup, it should be group_create**d**

</Section>

## Configuration

<Section>

<Text>

### `group_created`

> Informs that group has been created on license.

Reviewer: `group` -> `a group`
I'd also replace `license` with `a license` or `the license`, but not 100% sure - ask Ola :)

Reviewer 2: I try to avoid "on the license" in favor of "within a license".

> Informs that a group has been created on the license.

Reviewer 2: ```suggestion
Informs that a group has been created within the license.
```

#### Specifics

|                        |                                                                                                                                                                                       |
| ---------------------- | -------------------------------------------------------------------------------- |
| **Action**             | `create_group`                                                                   |
| **Push equivalent in** | [`Agent Chat API`](/messaging/agent-chat-api/v3.4/rtm-reference/#group_created)  |

</Text>

<Code>

<CodeResponse title={'Sample webhook'}>

</CodeResponse>

> ```json

Reviewer 2: Payloads should be put within the CodeResponse component, like so:

```
<CodeResponse title={'Sample webhook'}>

payload

</CodeResponse>
```

(with the proper indentation).

Currently, they're rendered as:

![image](https://user-images.githubusercontent.com/42515529/119127826-14354300-ba35-11eb-8a48-66fb30481302.png)

> | **Push equivalent in** | [`Agent Chat API`](/messaging/agent-chat-api/v3.4/rtm-reference/#group_created)  |

Reviewer: #opt What's up with the variable row length?

{
  "id": 42,
  "name": "sales",
  "language_code": "en",
  "agent_priorities": {
    "agent@example.com": "normal",
    "other_agent@example.com": "first"
  }
}
```

</Code>

</Section>

<Section>

<Text>

### `group_deleted`

> Informs that group has been deleted on license.

Reviewer 2: ```suggestion
Informs that a group has been deleted within a license.
```

>   }

Reviewer: Indentation is messed up - does this render correctly?

Author: it's rendering nice, but I'll fix it

Reviewer: If there are any more changes to apply - there's still a couple places that need indentation fixes

> Informs that group has been updated on license.  Only changed fields are sent.

Reviewer: Same remarks as with `group_created`.
Also, do we use the american double space to separate phrases?

#### Specifics

|                        |                                                                                                                                                                                       |
| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| **Action**             | `delete_group`                                                                                                                                                               |
| **Push equivalent in** | [`Agent Chat API`](/messaging/agent-chat-api/v3.4/rtm-reference/#group_deleted)  |

</Text>

<Code>

<CodeResponse title={'Sample webhook'}>

</CodeResponse>

```json
{
  "id": 42
}
```

</Code>

</Section>

<Section>

<Text>

### `group_updated`

> Informs that a group has been updated on the license. Only changed fields are sent.

Reviewer 2: ```suggestion
Informs that a group has been updated within a license. Only changed fields are sent.
```
